### PR TITLE
Feature: Human culture - Post-conference at the pub 

### PR DESCRIPTION
**Pull request by dorbarker**
## Summary
Adds a culture mission on urban human planets. The player can overhear snippets of conversation from a group of scientists who have gone to the pub following an academic conference.

Alternatively, instead of being implemented as a culture mission, it could be pared down and parts of it converted to a couple of `news` entries.

### CODE REVIEW

File: data/human/culture conversations.txt [hunk 1]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+		government "Republic"
+		attributes "urban"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`After a long journey and a longer-feeling landing procedure, you notice a group of people enter the pub where you're sitting. They stream in from the conference complex across the street. They vary in age, dress, and regional accent, but most have an identical badge dangling from a lanyard around their necks. You notice one that's been dropped on the floor.`
+			`The badge reads "Attendee - 35th Annual Pan-galactic Conference on Interstellar Microbial Genomics and Population Dynamics".`
+			`The group takes their seats a long table near to where you are sitting.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:540]:
```suggestion
			`As you take a break after a long journey and a longer-feeling landing procedure, you notice a group of people enter the pub where you're sitting, streaming in from the conference complex across the street. They vary in age, dress, and accent, but most have an identical badge dangling from a lanyard around their necks. You notice one badge that's been dropped on the floor, reading  "Attendee - 35th Annual Pan-galactic Conference on Interstellar Microbial Genomics and Population Dynamics". The group takes their seats at a long table near to you.`
```

File: data/human/culture conversations.txt [hunk 2]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+			choice
+				`(Order a meal and a drink for yourself.)`
+					goto "overheard scientists 1" 
+				`(Find somewhere quieter to spend your limited time planetside)`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:544]:
```suggestion
				`	(Find somewhere quieter to relax.)`
```

File: data/human/culture conversations.txt [hunk 3]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`After a long journey and a longer-feeling landing procedure, you notice a group of people enter the pub where you're sitting. They stream in from the conference complex across the street. They vary in age, dress, and regional accent, but most have an identical badge dangling from a lanyard around their necks. You notice one that's been dropped on the floor.`
+			`The badge reads "Attendee - 35th Annual Pan-galactic Conference on Interstellar Microbial Genomics and Population Dynamics".`
+			`The group takes their seats a long table near to where you are sitting.`
+			choice
+				`(Order a meal and a drink for yourself.)`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:542]:
```suggestion
				`	(Order a meal and listen in to their conversation.)`
```

File: data/human/culture conversations.txt [hunk 4]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+					decline
+			label "overheard scientists 1"
+			`Fragments of conversation make their way to you through the din of the pub:` 
+			` `
+			`"I'm so glad to have my presentation over with. I was so worried about having to present on the first day, but now I can finally just relax and actually listen to the other talks!"`
+			`"Oh definitely! My talk isn't until the third day. I probably shouldn't even be here - I should really be rehearsing." (She orders another drink anyway)`
+			`---`
+			`"Is there some law I don't know about that requires every 'question' to start with 'I guess this isn't so much a question as a comment...'?"`
+			`---`
+			`"I don't think the nucleotide evidence supports their hypothesis at all! Like, I know the proteomic data could be used to support their conclusion, but they're ignoring synonymous mutations!"`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:554]: Avoid using border lines - they look bad in the ES engine. Instead use descriptions to mark the passage of time. Also avoid using bracketed text outside of action choices (or side comments).

File: data/human/culture conversations.txt [hunk 5]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+			choice
+				`(Order another beverage and continue listening.)`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:556]:
```suggestion
				`	(Order another beverage and continue listening.)`
```

File: data/human/culture conversations.txt [hunk 6]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+					goto "overheard scientists 2"
+				`(Finish your meal and leave.)`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:558]:
```suggestion
				`	(Finish your meal and leave.)`
```

File: data/human/culture conversations.txt [hunk 7]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+					decline
+			label "overheard scientists 2"
+			`The group's conversation grows louder by virtue of both passion for the subject of conversation as well as the effects of a few drinks.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:561]:
```suggestion
			`	The group's conversation grows louder by virtue of both passion for the subject as well as the influence of a few drinks.`
```

File: data/human/culture conversations.txt [hunk 8]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+			`"Yeah, but I'm still terrified. I hate public speaking."`
+			`"Don't worry, you know about your particular area of expertise than they do. You'll be fine."`
+			`---`
+			`"These conferences only last three or four days, but it always feels like it ages you a year."`
+			`---`
+			`"Teaching sucks. Between class hours, office hours, and preparing lectures, there's hardly any time left for research."`
+			`"That's funny! Teaching undergrads is my one respite from grant-writing!"`
+			choice
+				`You leave after finishing your meal, reflecting on having overheard this slice of a life very different from your own.`
+					decline
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:588]:
```suggestion
			`	You leave after finishing your meal, reflecting on having overheard a slice of a life very different from your own.`
				decline
```

File: data/human/culture conversations.txt [hunk 9]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+			`"Can you believe my computer crashed right before I had to present? Good thing I had backups. Two is one and one is none."`
+			`---`
+			`"Singh and Esterhazy are such arrogant jerks. Their work is okay, I guess, but not enough to justify their wannabe rockstar personalities!"`
+			`"Sure, but they aren't just arrogant jerks, they're charismatic arrogant jerks. Charisma attracts good grad students to their lab, who produce good papers, who they put their name on. It's a positive feedback loop."`
+			---
+			`"The Republic Medical Research Agency said my work is too theoretical, but the University of Chiron Department of Microbiology says it's too applied!"`
+			`"That's a conundrum, alright. Maybe you could find some funding from industry? The Syndicate might be interested in your work."`
+			`"Yeah, but I'm not sure yet if I'm willing to give up my academic freedom."`
+			choice
+				`(Finish up at your own pace.)`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:573]:
```suggestion
				`	(Finish up at your own pace.)`
```

File: data/human/culture conversations.txt [hunk 10]
```diff
@@ -524,3 +524,65 @@ mission "News Reporter - Paradise"
...
+					goto "overheard scientists 3"
+				`(Settle your bill as quickly as possible and leave.)`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:575]:
```suggestion
				`	(Settle your bill as quickly as possible and leave.)`
```

**Review by Anarchist2 — COMMENTED** [review]
Overall, the dialogue is good, but it's formatted messily. Make sure to indent paragraphs after the first one in a conversation.

**Comment by dorbarker** [comment]
Thanks for the edits. I've accepted the changes for indentation and some rewording. 

Marking as draft for the moment while I rework some of the requested changes like removing the border lines. I'll re-request review once that's resolved.

**Review by Zitchas — COMMENTED** [review]
This is a fun little collection of bits of conversation from a conference. It looks like it fits fairly nicely. 

Anarchist2 does have a good point about replacing the `---` with descriptions to show the passage of time or breaks in the conversation. I'm not sure if doing so is strictly required, but it's definitely not something we have much of in ES so it does stand out a bit. As such, replacing it with descriptive text would help it mesh with the current game style.

**Comment by dorbarker** [comment]
These latest changes should resolve problems with the earlier version. Indentation should now follow the style guide (which I overlooked before - apologies!), and border lines have been replaced with short scene descriptions.

I also did some miscellaneous editing and reorganized some of the conversations.

File: data/human/culture conversations.txt [hunk 11]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+			choice
+				`	(Order a meal and listen in to their conversation.)`
+					goto "overheard scientists 1" 
+				`	(Find somewhere quieter to relax.)`
+					decline
+			label "overheard scientists 1"
+			`	Fragments of conversation make their way to you through the din of the pub.`
+			` `
+			`	Two of the younger attendees are looking a bit nervous.`
+			`	"I'm so glad to have my presentation over with. I was so worried about having to present on the first day, but now I can finally just relax and actually listen to the other talks!"`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:548]: Moving the description and first lines of dialogue into a single line; line breaks are meant to be after changes in speaker.
```suggestion
			`	Two of the younger attendees are looking a bit nervous. "I'm so glad to have my presentation over with. I was so worried about having to present on the first day, but now I can finally just relax and actually listen to the other talks!"`
```

File: data/human/culture conversations.txt [hunk 12]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+			`	"Oh definitely! I probably shouldn't even be here - I should really be rehearsing... Oh! Waiter, another beer, please!"`
+			`	"Got your slide deck ready for tomorrow?"`
+			`	"Yeah, but I'm still terrified. I hate public speaking."`
+			`	"Don't worry, you know more about your particular area of expertise than they do. You'll be fine once you're on stage! And besides, graduate school is one giant exercise in public speaking. Comfort comes with practice."`
+			` `
+			`	Another scientist is rolling his eyes after a post-mortem of the day's events.`
+			`	"Is there some law I don't know about that requires every 'question' to start with 'I guess this isn't so much a question as a comment...'?"`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:555]:
```suggestion
			`	Another scientist is rolling his eyes after a post-mortem of the day's events. "Is there some law I don't know about that requires every 'question' to start with 'I guess this isn't so much a question as a comment...'?"`
```

File: data/human/culture conversations.txt [hunk 13]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+			` `
+			`	An exasperated academic is speaking emphatically to her friend about what you infer to be a presentation they both heard earlier.`
+			`	"I don't think the nucleotide evidence supports their hypothesis at all! Like, I know the proteomic data could be used to support their conclusion, but they're ignoring synonymous mutations!"`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:558]:
```suggestion
			`	An exasperated academic is speaking emphatically to her friend about what you infer to be a presentation they both heard earlier. "I don't think the nucleotide evidence supports their hypothesis at all! Like, I know the proteomic data could be used to support their conclusion, but they're ignoring synonymous mutations!"`
```

File: data/human/culture conversations.txt [hunk 14]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+				`	(Order another beverage and continue listening.)`
+					goto "overheard scientists 2"
+				`	(Finish your meal and leave.)`
+					decline
+			label "overheard scientists 2"
+			`	The group's conversation grows louder by virtue of both passion for the subject as well as the influence of a few drinks.`
+			`	"Look, the analysis might be fine, but if they can't even be bothered to provide the version numbers for their analytical software, then it isn't reproducible and that's an automatic 'reject' from me."`
+			` `
+			`	One person is looking slightly shell-shocked after considering what might have been.`
+			`	"Can you believe my computer crashed right before I had to present? Good thing I had backups. Two is one and one is none."`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:569]:
```suggestion
			`	One person is looking slightly shell-shocked after considering what might have been. "Can you believe my computer crashed right before I had to present? Good thing I had backups. Two is one and one is none."`
```

File: data/human/culture conversations.txt [hunk 15]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+			` `
+			`	A group seem to be discussing some other colleagues in the field who are, presumably, not present this evening.`
+			`	"Singh and Esterhazy are such arrogant jerks. Their work is okay, I guess, but not enough to justify their wannabe rockstar personalities!"`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:572]:
```suggestion
			`	A group seem to be discussing some other colleagues in the field who are, presumably, not present this evening. "Singh and Esterhazy are such arrogant jerks. Their work is okay, I guess, but not enough to justify their wannabe rockstar personalities!"`
```

File: data/human/culture conversations.txt [hunk 16]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+			`	"Sure, but they aren't just arrogant jerks, they're charismatic arrogant jerks. Charisma attracts good grad students to their lab, who produce good papers, who they put their name on. It's a positive feedback loop."`
+			` `
+			`	Grant applications seem to be a common subject.`
+			`	"The Republic Medical Research Agency said my work is too theoretical, but the University of Chiron Department of Microbiology says it's too applied!"`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:576]:
```suggestion
			`	Grant applications seem to be a common subject. "The Republic Medical Research Agency said my work is too theoretical, but the University of Chiron Department of Microbiology says it's too applied!"`
```

File: data/human/culture conversations.txt [hunk 17]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+			`	"That's a conundrum, alright. Maybe you could find some funding from industry? The Syndicate might be interested in your work."`
+			`	"Yeah, but I'm not sure yet if I'm willing to give up my academic freedom."`
+			choice
+				`	(Finish up at your own pace.)`
+					goto "overheard scientists 3"
+				`	(Settle your bill as quickly as possible and leave.)`
+					decline
+			label "overheard scientists 3"
+			`	Two attendees compare gripes about job demands.`
+			`	"Teaching sucks. Between class hours, office hours, and preparing lectures, there's hardly any time left for research."`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:586]:
```suggestion
			`	Two attendees compare gripes about job demands. "Teaching sucks. Between class hours, office hours, and preparing lectures, there's hardly any time left for research."`
```

File: data/human/culture conversations.txt [hunk 18]
```diff
@@ -524,3 +524,69 @@ mission "News Reporter - Paradise"
...
+			`	"That's funny! Teaching undergrads is my one respite from grant-writing!"`
+			` `
+			`	A sociable, but exhausted-looking man is sitting back in his chair.`
+			`	"These conferences only last three or four days, but it always feels like it ages you a year."`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:590]:
```suggestion
			`	A sociable, but exhausted-looking man is sitting back in his chair. "These conferences only last three or four days, but it always feels like it ages you a year."`
```

**Review by Anarchist2 — COMMENTED** [review]
I'm not sure that the line spaces look good in-game; compare with and without spaces:

image

image

**Review by W1zrad — CHANGES_REQUESTED** [review]
Fun, funny, well-written! Very nice!

**Review by W1zrad — APPROVED** [review]
@Anarchist2 has a point about the extra line spacing being unusual, but I don't feel too strongly about it. If you plan to continue making missions (which I hope you do given the quality of this one), I'd say just try to refrain from the extra spaces in the future - but that's super minor. Thanks for the PR, enjoyed it!

**Pull request merged**